Code/Framework/AzCore/AzCore/Task/TaskGraph.h

```diff
@@ -16,6 +16,7 @@
 #include <AzCore/std/containers/vector.h>
 #include <AzCore/std/containers/unordered_map.h>
 #include <AzCore/std/parallel/binary_semaphore.h>
+#include <aZcore/RTTI/RTTI.h>
```

Comment:
Casing on "AzCore"

Reply:
thanks!!! seems I was not fast enough with SHIFT and then I was too fast... lol